config/domain_variations/Crawler_test.yaml

```diff
@@ -0,0 +1 @@
+resampling-interval: 5000
```

Comment:
Can we just not have these files for the environments that do not have reset parameters ?

Reply:
We won't be able to automatically do train/test on all environments without the files here. They are also placeholders, with the intention of being filled out once the environment refresh hits, and we can add the reset params to envs themselves.

Reply:
So there is an implicit contract that the file has to be present to be able to do automated training? I think it would be simpler, more future-proof to check for existence of the file rather than require all environment must have a sampler configuration.

Reply:
I agree. Like I said, I would not have them except for the fact that we *do* want to add parameters for these in the future. So I see no harm in including the empty files in the short term.